frontend/.vscode/launch.json

```diff
@@ -43,6 +43,16 @@
             },
             "cwd": "${workspaceRoot}/appflowy_flutter"
         },
+        {
+            "name": "AF-iOS: Build Dart Only",
```

Comment:
Added this task just for people who want to run AppFlowy, without rebuilding the backend